assets/app.css

```diff
@@ -67,3 +67,6 @@ div[role="main"] blockquote a:hover {
 
 /* We have to preset the div with left 0 for the animation to work */
 .wy-nav-content-wrap { left: 0%; }
+
+/* Links are hard to identify in paragraphs, so we underline them */
+.section a { text-decoration: underline; }
```

Comment:
Please move this to the top where `.section a` has been defined already (on line 6).

Furthermore, I would recommend to set the text colour to #404040 (because then it's the same colour as the paragraph, which also removes the odd looks of it and makes it better readable).

Reply:
alright, done that

Reply:
Perfect, thanks! :)